demo/src/test/java/org/gbif/pipelines/demo/hdfs/AvroToHdfsTestingPipelineTest.java

```diff
@@ -0,0 +1,74 @@
+package org.gbif.pipelines.demo.hdfs;
+
+import java.io.File;
+import java.net.URI;
+import java.util.Collections;
+
+import org.apache.beam.runners.direct.DirectRunner;
+import org.apache.beam.sdk.io.hdfs.HadoopFileSystemOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.FileUtil;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hdfs.MiniDFSCluster;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests the class {@link AvroToHdfsTestingPipelineTest}.
+ */
+public class AvroToHdfsTestingPipelineTest {
+
+  private MiniDFSCluster hdfsCluster;
+  private Configuration configuration = new Configuration();
+  private FileSystem fs;
+  private URI hdfsClusterBaseUri;
+
+  @Before
+  public void setUp() throws Exception {
```

Comment:
maybe this can be converted into a @BeforeClass annotation with all the implication of using static members only, if we do it we can avoid multiple instances of Hdsf minicluster for each test case